src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.big-field,
+.small-field {
+  display: block;
+  position: relative;
+}
+
+.input-big-field {
+  font-size: 16px;
+  padding: 0 62px;
+  height: 70px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.first-search-photo {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.input-small-field {
+  font-size: 14px;
+  padding: 0 33px;
+  height: 42px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
```

Comment:
here you duplicate styles in `input-big-field` and `input-small-field`. Pay attention to the checklist p.2 that I have mentioned before. It will help to avoid duplicates